gen_intake_gfdl.py

```diff
@@ -45,6 +46,13 @@ def main(inputdir,outputdir,filter_realm,filter_freq,filter_chunk,overwrite,appe
     project_dir = project_dir.rstrip("/")
     logger.info("Calling gfdlcrawler.crawlLocal")
     list_files = gfdlcrawler.crawlLocal(project_dir, dictFilter, dictFilterIgnore,logger)
+    with open("gfdl_test1.json", "r") as jsonFile:
+        data = json.load(jsonFile)
+
+    data["catalog_file"] = os.path.abspath(outputdir)
+
+    with open("gfdl_test1.json", "w") as jsonFile:
+        json.dump(data, jsonFile, indent=4)
```

Comment:
indent=4 is not reproducing the indentation gfdl_test1.json has. can you make sure it's consistent with the template? this is because at some point the automated tests will diff and check if it's similar, so it makes it easier.